dozer-types/src/types/field.rs

```diff
@@ -769,6 +772,240 @@ pub fn field_test_cases() -> impl Iterator<Item = Field> {
     .into_iter()
 }
 
+pub fn arrow_field_test_cases() -> impl Iterator<Item = Field> {
+    [
+        Field::UInt(0_u64),
+        Field::UInt(1_u64),
+        Field::Int(0_i64),
+        Field::Int(1_i64),
+        Field::Float(OrderedFloat::from(0_f64)),
+        Field::Float(OrderedFloat::from(1_f64)),
+        Field::Boolean(true),
+        Field::Boolean(false),
+        Field::String("".to_string()),
+        Field::String("1".to_string()),
+        Field::Text("".to_string()),
+        Field::Text("1".to_string()),
+        Field::Binary(vec![]),
+        Field::Binary(vec![1]),
+        Field::Timestamp(DateTime::from(Utc.timestamp_millis_opt(0).unwrap())),
+        Field::Timestamp(DateTime::parse_from_rfc3339("2020-01-01T00:00:00Z").unwrap()),
+        Field::Date(NaiveDate::from_ymd_opt(1970, 1, 1).unwrap()),
+        Field::Date(NaiveDate::from_ymd_opt(2020, 1, 1).unwrap()),
+        Field::Json(JsonValue::Array(vec![])),
+        Field::Json(JsonValue::Array(vec![
+            JsonValue::Number(OrderedFloat(123_f64)),
+            JsonValue::Number(OrderedFloat(34_f64)),
+            JsonValue::Number(OrderedFloat(97_f64)),
+            JsonValue::Number(OrderedFloat(98_f64)),
+            JsonValue::Number(OrderedFloat(99_f64)),
+            JsonValue::Number(OrderedFloat(34_f64)),
+            JsonValue::Number(OrderedFloat(58_f64)),
+            JsonValue::Number(OrderedFloat(34_f64)),
+            JsonValue::Number(OrderedFloat(102_f64)),
+            JsonValue::Number(OrderedFloat(111_f64)),
+            JsonValue::Number(OrderedFloat(111_f64)),
+            JsonValue::Number(OrderedFloat(34_f64)),
+        ])),
+        Field::Null,
+    ]
+    .into_iter()
+}
+
+pub fn schema_test_cases() -> Schema {
```

Comment:
Rename to `arrow_field_test_cases_schema`?

Reply:
sure